custom_components/plugwise_usb/__init__.py

```
@@ -186,12 +186,12 @@ async def async_remove_config_entry_device(
         and identifier[1] in (str(api_stick.mac_stick), str(api_stick.mac_coordinator))
     )
     if removable:
-        mac = device_entry.serial_number
+        mac = identifier[1]
```

Ruff (0.12.2)

Undefined name `identifier`

(F821)